kafkajobs/opts.go

```diff
@@ -119,3 +120,9 @@ type ConsumerOpts struct {
 	ConsumePartitions   map[string]map[int32]*Offset `mapstructure:"consume_partitions" json:"consume_partitions"`
 	ConsumerOffset      *Offset                      `mapstructure:"consumer_offset" json:"consumer_offset"`
 }
+
+type SSL struct {
```

Comment:
Would be better to use TLS, not SSL, since SSL is obsolete nowadays.

Reply:
Fixed